frontend/src/components/Cases/SchemaDisplay.tsx

```diff
@@ -21,29 +19,33 @@ const SchemaDisplay = ({
             <small className="font-monospace text-body-secondary text-uppercase">
               Schema
             </small>
-            <div className="d-flex ms-auto">
-              <CopyToClipboard textToCopy={schemaFormatted} />
-            </div>
+            <div className="d-flex ms-auto"></div>
           </div>
-          <div className="card-body">
-            <pre id="schema-code" style={schemaStyle}>
+          <div className="card-body position-relative">
+            <CopyToClipboard
+              textToCopy={schemaFormatted}
+              style="position-absolute top-0 end-0 mt-4 me-3"
+            />
+            <SyntaxHighlighter language="javascript" style={atomDark}>
```

Comment:
Uh, wait, I just noticed you have `language="javascript"` here which isn't right obviously -- surely there's a `language="json"` no?